pages/api/home/home.tsx

```diff
@@ -108,6 +106,8 @@ const Home = ({
     });
 
     saveConversation(conversation);
+
```

Comment:
When selecting a conversation from the sidebar, figured it would be nice if the chat input was ready to go.  Added it here in `handleSelectionConversation()`.